xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/ViewFile.xml

```diff
@@ -178,7 +178,77 @@ Full Presentation:
       <cache>long</cache>
     </property>
     <property>
-      <code>window.addEventListener("DOMContentLoaded", function () {
+      <code>require.config({
+  paths: {
+    'xwiki-suggestAttachments': "$xwiki.getSkinFile('uicomponents/suggest/suggestAttachments.js', true)" +
+    "?v=$escapetool.url($xwiki.version)"
+  }
+});
+
+require(['jquery', 'xwiki-suggestAttachments'], function($) {
+  const acceptedExtensions = '.ppt,.pptx,.odp,.doc,.docx,.odt,.xls,.xlsx,.ods,.pdf';
+  const styleObserver = new MutationObserver(function(mutations) {
```

Comment:
I'd add a comment to explain why is this needed, here or for other parts that are not so obvious. It makes sense in the context of this PR why all changes are needed, but it can be harder when someone tries to understand the reason. It will also help to mention that this `trick` will be removed once there is a proper way to do it in the platform (and link a jira issue)